presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -468,6 +469,7 @@ std::unique_ptr<TaskInfo> TaskManager::createOrUpdateTask(
   auto prestoTask = findOrCreateTask(taskId, startProcessCpuTime);
   {
     std::lock_guard<std::mutex> l(prestoTask->mutex);
+    prestoTask->updateCoordinatorHeartbeatLocked();
```

Comment:
Why we don't update updateCoordinatorHeartbeatLocked in findOrCreateTask? thanks!

Reply:
Because findOrCreateTask() is called from literally everywhere and we only need to cover the Coord's endpoints.